content/accessibility/context-menus.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Context menus
+---
+
+## Overview
```

Comment:
Would it be helpful to add a screenshot of a context menu?

Reply:
Please make sure the image follows the same style as other images: https://primer.style/design/accessibility/tooltip-alternatives

Reply:
@yaili it's okay, I've got this one ;)